src/thor/unidirectional_astar.cc

```diff
@@ -236,9 +236,8 @@ inline bool UnidirectionalAStar<expansion_direction, FORWARD>::ExpandInner(
 
   // Compute the cost to the end of this edge
   uint8_t flow_sources;
-  auto edge_cost = FORWARD ? costing_->EdgeCost(meta.edge, tile, time_info.second_of_week,
-                                                flow_sources, time_info.seconds_from_now)
-                           : costing_->EdgeCost(opp_edge, t2, time_info.second_of_week, flow_sources);
+  auto edge_cost = FORWARD ? costing_->EdgeCost(meta.edge, tile, time_info, flow_sources)
+                           : costing_->EdgeCost(opp_edge, t2, time_info, flow_sources);
```

Comment:
before we pass `seconds_from_now=0` for reverse search, but now we pass the whole `time_info`. what is the value of `time_info.seconds_from_now`

Reply:
I validated, that time is calculated correctly starting from the endpoint, and back to the origin. So it is a good option to use our upgraded traffic model. In case `arrive_by` is in 10 hours, we will not use live traffic. But when it is in 30 minutes speed will be interpolated both from `predicted` and `live`. ETA will be more correct in this case.